php_bson.h

```diff
@@ -71,7 +71,7 @@ typedef struct {
 } php_phongo_bson_state;
 
 #if PHP_VERSION_ID >= 70000
-#define PHONGO_BSON_STATE_INITIALIZER  { {}, { PHONGO_TYPEMAP_NONE, NULL, PHONGO_TYPEMAP_NONE, NULL, PHONGO_TYPEMAP_NONE, NULL}, NULL}
+#define PHONGO_BSON_STATE_INITIALIZER  { {{ 0 }}, { PHONGO_TYPEMAP_NONE, NULL, PHONGO_TYPEMAP_NONE, NULL, PHONGO_TYPEMAP_NONE, NULL}, NULL}
```

Comment:
@derickr: See http://stackoverflow.com/q/11551986 and http://stackoverflow.com/a/20405554. Windows required `{ 0 }` (as is already done in libmongoc), but the extra braces here are a concession for gcc's missing-braces warning. Windows seems to be OK with it, too, so I gathered this was preferable to disabling the warning altogether.